adcc/LazyMp.py

```diff
@@ -22,11 +22,24 @@
 ## ---------------------------------------------------------------------
 import libadcc
 import numpy as np
+import textwrap
 
-from .mp import compute_mp2_diffdm  # noqa: F401
-from .misc import cached_property
+from .functions import direct_sum, evaluate, einsum
+from .misc import cached_property, cached_member_function
 from .ReferenceState import ReferenceState
 from .OneParticleOperator import OneParticleOperator, product_trace
+from .Intermediates import register_as_intermediate
+from .timings import Timer, timed_member_call
+from . import block as b
+
+
+def split_spaces(spacestr):
+    sp = textwrap.wrap(spacestr, 2)
+    try:
+        assert all(s in ('o1', 'o2', 'v1') for s in sp)
```

Comment:
(a) I think this is more general and should live in a util / common type file (maybe block?)
(b) what about the other spaces (o3, v2, v3 etc). Should not be too hard to support this.

Reply:
It used to be in `block`, but then the python 3.6 were failing, because when you say `b.split_spaces`, it dispatches to `__getattr__` and then fails 😄 I think `block` would be the right place though.

Reply:
Ah yes, that's true. Maybe just split_spaces.py ?

Reply:
Or in MoSpaces.py? That would make sense, too.

Reply:
Yes, ok, that's fine with me.